### PR TITLE
changes made in buttons and subscribe button

### PR DESCRIPTION
**Pull request by Cypher1002**
## 🛠️ Fixes Issue

Closes #186

## 👨‍💻 Changes proposed
changes made in buttons in counter section
added animations and effects 
change made in subscribe button removed border

## ✔️ Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
i have done the changes accordingly and if any more changes are required will do it asap

## 📷 Screenshots
updated-counter_section

### CODE REVIEW

**Comment by The-Shivam-garg** [comment]
please resolve the PR conflicts

**Comment by sakshamgurbhele** [comment]
@Cypher1002 kindly resolve the conflicts!

File: css/main.css [hunk 1]
```diff
@@ -12,6 +13,18 @@
...
+  --cyan: #12a4d9;
+  --black: #0e0d0d;
+  --lightBlack: #a29a9c;
+  --exLight: #f4f3f3;
+  --transition: all 0.4s ease-in-out;
+  --Heebo: "Heebo", sans-serif;
+  --Roboto: "Robot", sans-serif;
+}
+ main
+
```
Comment (sakshamgurbhele) [inline css/main.css:27]: 👍🏼

File: css/main.css [hunk 2]
```diff
@@ -237,12 +250,17 @@ html {
     padding-bottom: 0.4rem;
     padding: 0.5rem;
     background-image: var(--Color);
+ margin: 0.4rem 0;
+  color: white;
+  border-radius: 0.3rem;
+  padding-bottom: 1rem;
```
Comment (sakshamgurbhele) [inline css/main.css:256]: 👍🏼

File: css/main.css [hunk 3]
```diff
@@ -266,6 +291,13 @@ html {
 .counter .num {
     font-size: 3.2rem;
     font-weight: bolder;
+.counter .item:nth-child(4) {
+  background-image: linear-gradient(thistle, darkslategrey);
+}
+
+.counter .num {
+  font-size: 2.6rem;
+  font-weight: 900;
```
Comment (sakshamgurbhele) [inline css/main.css:300]: 👍🏼

File: css/main.css [hunk 4]
```diff
@@ -322,12 +354,18 @@ html {
...
     width: 20rem;
     font-size: 1.7rem;
     border-radius: 5rem;
     cursor: pointer;
     border: none;
+  background: #fff;
+  width: 20rem;
+  font-size: 1.7rem;
+  border-radius: 5rem;
+  cursor: pointer;
```
Comment (sakshamgurbhele) [inline css/main.css:368]: 👍🏼

**Review by sakshamgurbhele — APPROVED** [review]
LGTM 👍🏼 🥇

**Comment by Cypher1002** [comment]
sir @sakshamgurbhele are my changes merged to the main? what is status could you give me some info about it?

**Pull request merged**